ludwig/datasets/configs/goodbooks_books.yaml

```diff
@@ -13,3 +13,4 @@ description: |
     book metadata (author, year, etc.)
     tags/shelves/genres
   https://github.com/zygmuntz/goodbooks-10k
+task: Multi-Class Classification
```

Comment:
Is "Multi-Class Classification" different from "Classification"?

Reply:
It's just a more specific way of saying it's a classification problem. Since there are a bunch of binary classification problems in the zoo, I figured I would distinguish between those with this label.

Reply:
This is my assumption I make (I wanted to check that this is correct):

"Multi-Class Classification" - 3 or more classes
"Binary Classification" - 2 classes
"Classification" - 2 or more classes

Lets document what these mean in the Readme.md, docstring of the `types` field, or both.